### PR TITLE
Fix a couple of rst warnings

### PR DESCRIPTION
**Pull request by markvl**

### CODE REVIEW

File: docs/scenarios/gui.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ and the mailing list https://groups.google.com/forum/#!forum/project-camelot
 
 Cocoa
 -----
-.. note:: The Cocoa framework is only available on OS�X. Don't pick this if you're writing a cross-platform application!
+.. note:: The Cocoa framework is only available on OS |_| X. Don't pick this if you're writing a cross-platform application!
```
Comment (sigmavirus24) [inline docs/scenarios/gui.rst]: Why do we need that unicode character there? Why can't it just be a space?

File: docs/shipping/freezing.rst [hunk 2]
```diff
@@ -107,8 +107,8 @@ Prerequisite is to have installed :ref:`Python, Setuptools and pywin32 dependenc
 - `Manual <http://www.pyinstaller.org/export/d3398dd79b68901ae1edd761f3fe0f4ff19cfb1a/project/doc/Manual.html?format=raw>`_
 
 
-OS�X
-----
+OS |_| X
```
Comment (sigmavirus24) [inline docs/shipping/freezing.rst]: Again, I'm pretty sure we don't care about a special unicode character here.

**Comment by sigmavirus24** [comment]
Please ping me when you've fixed those two items up.

**Comment by markvl** [comment]
I wanted to remove the warning without changing what was there too much. I assumed the unicode "non breaking space" was approved (required?) somewhere in the past for a reason.

@sigmavirus24 I have updated the first commit to remove the character completely.

**Comment by sigmavirus24** [comment]
I don't know if they were approved or required. They were probably added without anyone noticing because GitHub doesn't display them in a way such that one might notice it. I'm guessing it was purely accidental and not intentional.

Thanks for fixing this up @markvl

**Pull request merged**